index.html

```diff
@@ -7,33 +7,145 @@
+                    <button class="close-mobile absolute left-4 top-2"><i class="las la-times"></i></button>
+                    <li class="close-mobile w-full py-10 border-y-2 text-center"><a href="about.html" class="highlight text-sm">About</a></li>
+                    <li class="close-mobile w-full py-10 border-b-2 text-center"><a href="#main-program" class="text-sm">Program</a></li>
+                    <li class="close-mobile w-full py-10 border-b-2 text-center"><a href="#" class="text-sm">Join</a></li>
+                    <li class="close-mobile w-full py-10 border-b-2 text-center"><a href="#partners" class="text-sm">Sponsor</a></li>
+                    <li class="close-mobile w-full py-10 text-center"><a href="#" class="text-sm">News</a></li>
+                </ul>
+            </div>
+        </div>
     </header>
+
+    <section id="hero" class="bg-[url('img/pexels-tara-winstead-8386422.jpg')] bg-cover">
+        <span class="sr-only">An Android and an human reaching each other's hand</span>
+        <div class="pt-8 pb-14 px-2 bg-[#dfdfdf]/[.85] md:pt-16 md:pb-20 md:px-24 lg:h-[90vh] lg:px-32 lg:flex lg:flex-col lg:justify-center xl:px-60">
+            <div class="heading px-1 md:px-0">
+                <p class="highlight text-xl md:text-2xl lg:mb-3 lg:text-3xl xl:text-5xl">Think. Build. Share</p>
+                <h1 class="highlight text-2xl font-bold uppercase md:text-3xl lg:text-[2.5rem] xl:text-7xl">
+                    AI quotidian applications
+                    <span class="block highlight">2023 Summit</span>
+                </h1>
+            </div>
+        
+            <div class="border-2 border-white my-4 py-3 px-4 md:w-3/4 lg:w-3/5 xl:w-1/2 xl:py-5">
+                <p class="text-sm font-semibold md:text-xs md:font-normal lg:text-sm xl:text-base">A celebration to share the positive changes that
+                    everyday AI implementations could bring, taking place
+                    this September in London.</p>
+            </div>
+        
+            <div class="heading px-1 tracking-wide md:pt-1 md:px-0">
+                <h2 class="text-lg font-bold md:text-xl lg:mb-1 lg:text-2xl xl:mb-2 xl:text-3xl xl:font-black">2023.09.16 (SAT) - 17 (SUN)</h2>
+                <h3 class="xl:text-xl">@The Natural History Museum</h3>
+            </div>
+        </div>
+    </section>
     
+    <section id="main-program" class="pt-8 bg-[#191d25] text-center md:pb-12 lg:pb-16 xl:min-h-[60vh]">
+        <h2 class="text-xl text-white font-semibold xl:text-3xl">Main Program</h2>
+        <div class="flex justify-center">
+            <i class="las la-minus text-2xl text-[#ec5242]"></i>
+            <i class="las la-minus text-2xl text-[#ec5242] ml-[-0.5rem]"></i>
+        </div>
+    
+        <div class="program-grid mt-3 px-3 flex flex-wrap gap-2 md:px-12 md:justify-center md:gap-1 md:flex-nowrap lg:px-36 xl:mt-6 xl:px-60">
```

Comment:
Good job with wrapping semantic section element. Let's wrap our card with semantic article tag to make it more semantic and SEO friendly.